api/v2/serializers/details/allocation_source.py

```diff
@@ -63,6 +63,6 @@ def get_updated(self, allocation_source):
     class Meta:
         model = AllocationSource
         fields = (
-            'id', 'url', 'name', 'source_id', 'compute_allowed',
```

Comment:
I asked a similar question, I think that [Troposphere PR 566](https://github.com/cyverse/troposphere/pull/566) addresses the change from `..source_id` to `..name`. 

The only curious thing is that Connor has a fix for an issue with `SENTRY_ENABLED` on that _fixes_ branch that we haven't pulled into W-W (see here: [f279cd407](https://github.com/cdosborn/troposphere/commit/f279cd4076a1ecb2dc40ee2c3572ebf8c8bd675e)).